packages/react-components/review/react-components.api.md

```diff
@@ -371,6 +374,49 @@ export interface DevicesButtonStyles extends ControlBarButtonStyles {
     menuStyles?: Partial<DevicesButtonContextualMenuStyles>;
 }
 
+// @beta
+export const Dialpad: (props: DialpadProps) => JSX.Element;
+
+// @beta
+export interface DialpadButtonsType {
+    // (undocumented)
+    primaryContent: string;
+    // (undocumented)
+    secondaryContent?: string;
+}
+
+// @beta
+export interface DialpadProps {
+    // (undocumented)
+    dialpadButtons?: DialpadButtonsType[][];
+    // (undocumented)
+    strings?: DialpadStrings;
+    // (undocumented)
+    styles?: Partial<DialpadStyles>;
```

Comment:
If each style in the interface is optional, is the `Partial` needed here?

Reply:
removed partial